deps/src/ideals.cpp

```diff
@@ -405,6 +416,7 @@ void singular_define_ideals(jlcxx::Module & Singular)
   Singular.method("id_res", &id_res_helper);
   Singular.method("id_mres_map", &id_mres_map_helper);
   Singular.method("id_prune_map", &id_prune_map_helper);
+  Singular.method("id_prune_map_v", &id_prune_map_v_helper);
```

Comment:
But what does this do and which code is going to call it?

Reply:
id_prune_map and id_prune_map_v are 2 interfaces to computing a minimal embedding of the given module (in Singular). Return the minimal embedding, the map, and (in the 2nd case) an array how to map indices.
Should be documented and tested by the Singular.jl routines prune_map   and (coming) prune_map_v